src/components/Home.js

```diff
@@ -0,0 +1,44 @@
+import { useSelector } from 'react-redux';
+import CountUp from 'react-countup';
+import { FcGlobe } from 'react-icons/fc';
+import Spinner from './Spinner/Spinner';
+import ContinentList from './ContinentList';
+
+const Home = () => {
+  const {
+    continents,
+    totalData: { totalCases, totalDeaths },
+  } = useSelector((state) => state.covid);
+
```

Comment:
- [ ] In the home component it is expected of us to add a stateless filter component that can be used to filter the data on the homepage.

  This was hinted at in the **_`Description`_** of the project. See the quotation below:
  
  ---
  > one page with a list of items that could be filtered by some parameters; in the example below, it's a list of metrics that can be filtered by the country (imagine a search field to introduce the country name like Italy, Croatia, etc.). This page should be your homepage.
  ---
  
  Then, under **_`Interactions`_** section, under the sub-title **_`Home page`_**, this requirement is mentioned. See quote below:
  
  ---
  ### Home page
  - When the page loads, the web app shows the list of categories that could be filtered out by some parameter, for example by the category name.
  ---
  
  
  ### Explanation
  
  Here is what is meant by filtering:
  
  Think of a scenario where your app loads thousands of items on the homepage for the user to select, it would be tedious to scroll through all that to get to the particular one the user is interested in. Therefore, it would be nice to have something like a search field, in which the user can type the name of the item he/she is interested in. The app will then filter the items based on what the user types and then only display that item on the homepage for the user to select before navigating to the details page.
  
  Please let's work on how we can implement filtering of data that we find on the homepage.

Reply:
Filter implemented